src/miner_critical_sup.erl

```diff
@@ -105,7 +105,10 @@ init(_Opts) ->
         end,
 
 
-    ChildSpecs0 = [?SUP(blockchain_sup, [BlockchainOpts])] ++ ConsensusMgr,
+    ChildSpecs0 = [
```

Comment:
Probably belongs in the `restart_sup` don't you think?

Reply:
it doesn't actually; failures in the critical_sup will stop the whole system, so this definitely belongs in the restart_sup.

Reply:
I meant the metrics functionality works the same either way